inbm/dispatcher-agent/dispatcher/fota/installer.py

```diff
@@ -78,7 +78,7 @@ def get_product_params(self, platform_product: str) -> Dict:
                 if ' ' in tool_name or tool_name.isspace():
```

Comment:
<h3>$\color{#FFA500}Improvements: Code Refactoring$</h3>

The condition `if ' ' in tool_name or tool_name.isspace():` could be simplified to `if tool_name.isspace():` since `isspace()` returns True if there are only whitespace characters in the string.